### DISCUSSION TITLE
我看貌似有windows客户端，官网文档是不是应该更新了？

### DISCUSSION THREAD

**Discussion by gcxfd** [juicedata/juicefs#353] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/69608605/116206011-eda32580-a770-11eb-9707-337873c255dd.png)
见截图
https://github.com/juicedata/juicefs/blob/main/docs/en/windows.md

**Comment by davies** [comment, marked as answer]
官方文档是针对 商业版的，目前商业版的 Windows 客户端还没正式发布，目前还只有社区版的 Windows 客户端。

**Reply by yuhr123** [reply]
JuiceFS 社区版用户，请参考文档 [Windows 系统使用 JuiceFS](https://github.com/juicedata/juicefs/blob/main/docs/zh_cn/juicefs_on_windows.md)
